### ISSUE TITLE
Moran Scatterplot: Randomization not working in Windows

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lixun910/geoda#206]
```
GeoDa 1.7.45
Randomization not working in Windows 7 for any permutations (blank view, see 
attached).

```

Original issue reported on code.google.com by `jkoschin...@gmail.com` on 27 Jul 2015 at 12:35

Attachments:
* [moran scatterplot bug win.png](https://storage.googleapis.com/google-code-attachments/geoda/issue-206/comment-0/moran scatterplot bug win.png)

**Comment by lixun910** [comment]
fix is applied in 1.7.49

**Comment by jkoschinsky** [comment]
Verified fix in GeoDa 1.7.89 (Win 7 Professional (64-bit)).
